js/base-jsload.js

```diff
@@ -38,11 +38,12 @@ function baseJsLoad () {
 
 		// Monkey patch JSON loading
 		const cached = DataUtil.loadJSON;
-		DataUtil.loadJSON = async (url, ...others) => {
+		DataUtil.loadJSON = async (url, {isForceExternal=false} = {}, ...others) => {
```

Comment:
come to think of it, why are we even passing this in? Could we not just read `d20plus.debug.forceExternalRequests` inside this method and have a 2-line PR 🤔

Reply:
I thought it would allow things to be a bit more modular, but you're probably right. I'll change that.